angular/core/components/dashboard_page/dashboard_page.haml

```diff
@@ -31,11 +31,19 @@
     %h1.lmo-h1-medium.dashboard-page__heading{translate: "dashboard_page.filtering.all", ng-show: "dashboardPage.filter == 'show_all'"}
     %h1.lmo-h1-medium.dashboard-page__heading{translate: "dashboard_page.filtering.participating", ng-show: "dashboardPage.filter == 'show_participating'"}
     %h1.lmo-h1-medium.dashboard-page__heading{translate: "dashboard_page.filtering.muted", ng-show: "dashboardPage.filter == 'show_muted'"}
-    .dashboard-page__no-threads{ng-hide: "dashboardPage.loadMoreExecuting || dashboardPage.currentBaseQuery.any()"}
+    .dashboard-page__no-threads{ng-hide: "dashboardPage.loadMoreExecuting || dashboardPage.currentBaseQuery.any() || dashboardPage.noGroups()"}
       %span{ng-show: "dashboardPage.filter == 'show_all'", translate: "dashboard_page.no_threads.show_all"}>
       %span{ng-show: "dashboardPage.filter == 'show_participating'", translate: "dashboard_page.no_threads.show_participating"}>
       %span{ng-show: "dashboardPage.filter == 'show_muted'", translate: "dashboard_page.no_threads.show_muted"}>
       %a{ng-show: "dashboardPage.filter != 'show_all'", translate: "dashboard_page.view_recent", href: "", ng-click: "dashboardPage.setFilter('show_all')"}>
+    .dashboard-page__no-groups{ng-show: "dashboardPage.noGroups()"}
```

Comment:
They're not duplicates - they're messages for when you have no groups, the spans are messages for when you have no threads.